bundle/jsky.app.ot/src/main/scala/edu/gemini/catalog/ui/package.scala

```diff
@@ -70,8 +70,9 @@ case class ObservationInfo(ctx: Option[ObsContext],
                            offsets: Set[Offset],
                            catalog: CatalogName,
                            mt: MagnitudeTable) {
-  def catalogQuery:List[CatalogQuery] = validStrategies.collect {
-      case SupportedStrategy(s, query, _) if s == strategy => query
+  def catalogQuery: List[CatalogQuery] =
+    validStrategies.collect {
+      case SupportedStrategy(s, query, _) if strategy.map(_.strategy).contains(s) => query
```

Comment:
An added bonus here.  The `s == strategy` comparison we used before is an apples/oranges comparison of two different things that can never be equal.

Reply:
Nice